### PR TITLE
[FIX] Update control property

### PR DESCRIPTION
**Pull request by Chariyski**
After an error from setting a new value of a control property, the value should return back to the initial one.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/SAP/ui5-inspector/pull/19" target="_blank">ReviewNinja

File: app/scripts/injected/main.js [hunk 1]
```diff
@@ -130,18 +130,24 @@ sap.ui.require(['ToolsAPI'], function (ToolsAPI) {
...
-                control['set' + property](newValue);
 
-                // Update properties and bindings
+            try {
+                if (control) {
+                    // Change the property through its setter
+                    control['set' + property](newValue);
+                }
+            } catch (error) {
+                // Update properties and bindings after an error
```
Comment (matz3) [inline app/scripts/injected/main.js]: I've added this to always update the devtool with the actual value which might not be the same as handed over, e.g. "null" => property default value. So imo this should also be done when the actual update succeeds.

Comment (Chariyski) [inline app/scripts/injected/main.js]: Now it should be OK.

**Comment by matz3** [comment]
Nice :+1:

**Pull request merged**